### PR TITLE
Split coders into files

### PR DESCRIPTION
**Pull request by amomchilov**
Stemmed from this PR/conversation: https://github.com/trilemma-dev/SecureXPC/pull/13/files#r746223943

I made the fewest changes possible these files, to reduce bugs (since the diffs on this PR won't be very insightful).

We can do any further massaging and improvements in a follow up

### CODE REVIEW

**Comment by amomchilov** [comment]
Damn it, I need to edit these commits to replace tabs back to spaces (I wish Xcode had per-project preferences for tabs vs spaces)

File: Sources/SecureXPC/XPC Coders/xpc_type_t+description.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
+//
+//  File.swift
```
Comment (jakaplan) [inline Sources/SecureXPC/XPC Coders/xpc_type_t+description.swift:2]: Should be "xpc_type_t+description.swift"

**Review by jakaplan — APPROVED** [review]
LGTM. Both requested changes are really minor (and not changes the code itself), so I'll just merge this and make them myself.

**Comment by jakaplan** [comment]
> Damn it, I need to edit these commits to replace tabs back to spaces (I wish Xcode had per-project preferences for tabs vs spaces)

Yeah I noticed the other day you use tabs while I use spaces. Don't want to start a holy war here, so figured we can just resolve this later. Probably run a linter/formatter over the whole codebase at some point.

**Comment by amomchilov** [comment]
I’m okay with using spaces, but remembering to switch my Xcode settings between my projects is super hard lol (especially because I’m extending SecureXPC as I’m building out my main app).

A linter would be ideal

**Pull request merged**